dev_guide/secrets.adoc

```diff
@@ -12,11 +12,12 @@ toc::[]
 == Overview
 
 The `Secret` object type provides a mechanism to hold sensitive information such
-as passwords, OpenShift client config files, `dockercfg` files, etc. Secrets
-decouple sensitive content from the pods that use it and can be mounted into
-containers using a volume plug-in or used by the system to perform actions on
-behalf of a pod. This topic discusses important properties of secrets and
-provides an overview on how developers can use them.
+as passwords, OpenShift client config files, `dockercfg` files, combinations of an
+sername and password, tokens, private SSH keys, etc. Secrets decouple sensitive 
```

Comment:
I would simply say "private source repository credentials" instead of listing all the types. It might even be good to link to the section in the builds.adoc where they are discussed.